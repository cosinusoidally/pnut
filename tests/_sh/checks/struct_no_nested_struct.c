// expect_comp_failure

struct A {
  struct B {
    int a;
    int b;
  } b;
};

void main() {
  return;
}
