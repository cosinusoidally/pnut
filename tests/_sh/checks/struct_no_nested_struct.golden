tests/_sh/checks/struct_no_nested_struct.c:9:0  Nested structures not supported by shell backend. Use a reference type instead.
